test/pystorm/test_bolt.py

```diff
@@ -201,7 +201,7 @@ def test_process_tick(self, process_tick_mock, read_tuple_mock):
         # Make sure we send sync for heartbeats
         read_tuple_mock.return_value = Tuple(id=None, task=-1,
                                              component='__system',
-                                             stream='__tick', values=[50])
+                                             stream='__tick', values=(50))
```

Comment:
Nice catch! Lack of a trailing comma is one of the easiest syntax mistakes to make in Python.